andela/travel_tool_back
# #160370638 Implement delete comment functionality

#### What does this PR do?
This PR adds an endpoint to enable users delete comments

#### Description of Task to be completed?

- Add test cases for delete comment endpoint
- Update Comment schema with deletedAt field
- Create delete comment endpoint

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory -` cd travel_tool_back`
- Install project dependencies using `yarn install`
- Run database migrations using `yarn run db:migrate`
- Seed the database using `yarn run db:seed`
- Start the app using make start
- Navigate to [travela](http://travela-staging.andela.com/) in your browser and login
- Open the developer tools, locate applications tab, copy `jwt-token` in the cookies
- Check `.env.example` and set up your `.env` appropriately
- Get back to your command line, Install dependencies and start the server - `make start`
- In`Postman`, use the `jwt-token` to send a request by adding a `Header`, key `Authorization` and value the `token`.
- In Postman, in the `x-www-form-urlencoded` body options;
    - [x] Create a new request
    - [x] Add a new comment to the request created
- You should be able to delete the comment on `localhost:5000/api/v1/comments/:commentId`
- The response should include a message and a success flag

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160370638](https://www.pivotaltracker.com/story/show/160370638)

#### Screenshots (if appropriate)
N/A

#### Questions:
<img width="1438" alt="screen shot 2018-09-19 at 12 54 17 pm" src="https://user-images.githubusercontent.com/32720508/45751935-d6404b00-bc0b-11e8-9751-e09f89805b74.png">
